modules/prebidServerBidAdapter/index.js

```diff
@@ -728,6 +730,13 @@ Object.assign(ORTB2.prototype, {
       if (floor) {
         imp.bidfloor = floor.floor;
         imp.bidfloorcur = floor.currency;
+
+        const ortb2ImpFloorMin = imp.ext?.prebid?.floors?.floorMin;
```

Comment:
this should fall back to `imp.ext?.prebid?.floorMin` (see https://github.com/prebid/Prebid.js/issues/8938#issuecomment-1234653789);

also, if we are condiering the currency, this should be converted to `floorMinCur` before the comparison with `floorMin`.

Reply:
thanks, just pushed a new commit with the changes